src/CoreText/Adapter.cs

```diff
@@ -116,6 +116,11 @@ public static string GetStringValue (IDictionary<NSObject, NSObject> dictionary,
 			return ((NSNumber) value).UInt32Value;
 		}
 
+		public static bool? GetBoolValue (NSDictionary dictionary, NSObject key)
+		{
+			return CFBoolean.GetValue (dictionary.LowlevelObjectForKey (key.Handle));
```

Comment:
This can use NSNumber too (in fact it must, because CFBoolean returns a `bool`, not a `bool?`, which means it doesn't handle inexistent keys as expected).